### PR TITLE
Remove positive from hfsifrazil

### PR DESCRIPTION
**Pull request by xylar**
## Description

This fixes the issue mentioned in https://github.com/E3SM-Project/e3sm_to_cmip/pull/270#issuecomment-2339303713

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

If applicable:

- [ ] New and existing unit tests pass with my changes (locally and CI/CD build)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have noted that this is a breaking change for a major release (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: e3sm_to_cmip/cmor_handlers/mpas_vars/hfsifrazil.py [hunk 1]
```diff
@@ -74,28 +79,39 @@ def handle(infiles, tables, user_input_path, **kwargs):
...
+        {
+            "table_entry": "longitude",
+            "units": "degrees_east",
+            "coord_vals": ds.lon.values,
+            "cell_bounds": ds.lon_bnds.values,
+        },
+    ]
     try:
-        mpas.write_cmor(axes, ds, VAR_NAME, VAR_UNITS, positive='down')
+        mpas.write_cmor(axes, ds, VAR_NAME, VAR_UNITS)
```
Comment (xylar) [inline e3sm_to_cmip/cmor_handlers/mpas_vars/hfsifrazil.py:114]: This is the only "real" change here.  The rest is linting (and in its own commit).

**Comment by xylar** [comment]
@TonyB9000, I don't have the means to test this so I'll count on you.

**Comment by TonyB9000** [comment]
@xylar Looks Good.  I just re-ran CMIP6.CMIP.E3SM-Project.E3SM-2-1.1pctCO2.r1i1p1f1.Omon.hfsifrazil.gr (1 year) and the cmor log output no longer has the warning:

Warning: variable 'hfsifrazil' (table Omon) you passed positive value:down, but table does not mention it, will be ignored, if you really want this in your variable output use cmor_set_variable_attribute_internal function.

Next Steps?  Do you "squash and merge"?  Do I?  Does it matter? (my ignorance is showing).

**Comment by xylar** [comment]
You or @chengzhuzhang should merge, as I don't know the policies for this repo.

**Comment by TonyB9000** [comment]
@xylar Policies?  Ha!  (merging and deleting branch.)

**Comment by TonyB9000** [comment]
@xylar I guess only you can delete the branch - I get no opportunity.

**Comment by xylar** [comment]
@TonyB9000, I'd have to make you a collaborator on my fork to allow you to delete my branches.  That's probably more trouble than it's worth.  I can delete them.

**Comment by TonyB9000** [comment]
@xylar No issue.  I am just in the habit of deleting closed branches to reduce the clutter.  This is my first time dealing in someone else's fork - so I am ignorant of the protocols.

**Pull request merged**